packages/opentelemetry-instrumentation-xml-http-request/src/xhr.ts

```diff
@@ -171,6 +178,24 @@ export class XMLHttpRequestInstrumentation extends InstrumentationBase<XMLHttpRe
     }
   }
 
+  private _applyAttributesAfterXHR(span: api.Span, xhr: XMLHttpRequest) {
+    const applyCustomAttributesOnSpan = this._getConfig()
+      .applyCustomAttributesOnSpan;
+    if (applyCustomAttributesOnSpan) {
```

Comment:
Just to be a little safer as long as we're running a check

```suggestion
    if (typeof applyCustomAttributesOnSpan === 'function') {
```